core/schema_utils.py

```diff
@@ -416,6 +416,21 @@ def has_length_at_most(obj: List[str], max_value: int) -> bool:
         """
         return len(obj) <= max_value
 
+    @staticmethod
+    def has_length(obj: List[str], value: int) -> bool:
+        """Returns True iff the given object (a list) has exact
+        `value` elements.
+
+        Args:
+            obj: list(str). A list of strings.
```

Comment:
Are we sure that the list of elements in this object is always a string?

Reply:
I used signature of `has_length_at_least` and `has_length_at_most` for reference.

![Screenshot from 2021-10-05 14-16-28](https://user-images.githubusercontent.com/42772952/135991053-439be44b-36cb-48cf-a4ba-e62dfbe54ae8.png)